src/AvPortalResource.php

```diff
@@ -11,6 +11,12 @@
  */
 class AvPortalResource {
 
```

Comment:
FYI guys, all of this will be fixed if you use the `"openeuropa/code-review": "49-use-phpcsfixer"` branch.
It will be committed soon, but in the meantime, it can help you to fix all these annoying issues.
Don't forget to not commit the composer.json files with that branch!